.travis.yml

```diff
@@ -12,6 +12,10 @@ addons:
       - libboost-all-dev
       - libxml2-dev
       - libsnappy-dev
+stages:
+  - test
+  - name: deploy
+    if: tag =~ .*
```

Comment:
Does yaml support comments? Would be nice to explain this monster

Reply:
https://docs.travis-ci.com/user/conditional-builds-stages-jobs/#Conditional-Builds

Reply:
Nope, this won't do. You need to explain it with normal words.

Reply:
Anyway, I add a comment, see new commit below.